.github/workflows/test-code-code2parquet.yml

```diff
@@ -0,0 +1,95 @@
+#
+# DO NOT EDIT THIS: it is generated from test-transform.template,  Edit there and run make to change these files
+#
+name: Test - transforms/code/code2parquet
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "dev"
+            - "releases/**"
+        tags:
+            - "*"
+        paths:
+            - "transforms/code/code2parquet/**"
+            - "data-processing-lib/**"
```

Comment:
"kfp/kfp_support_lib/\**" and "kfp/kfp_ray_components/**"

Reply:
I guess this is to get tests run in */kfp_ray directories?  This makes sense, although the test-src and test-image targets in kfp_ray/Makefile seem to be noops.  If you're suggesting this to get the images republished, I'm wondering if image publishing should be done in a separate workflow?

Reply:
We will run kfp tests separately and later add kfp tests to the test-transform.template to include kfp testing only when the transform kfp_ray files change